sdk/tools/chkreg/chkreg.c

```diff
@@ -0,0 +1,132 @@
+/*
+ * PROJECT:     ReactOS Tools
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Check Registry Utility
+ * COPYRIGHT:   Copyright 2023 George Bișoc <george.bisoc@reactos.org>
+ */
+
+/* INCLUDES *****************************************************************/
+
+#include "chkreg.h"
+
+/* FUNCTIONS ****************************************************************/
+
+static
+void
+ChkRegShowUsage(
+    void)
+{
+    printf("Usage: chkreg /h /a /r /v HIVE_FILE LOG_FILE\n\n"
+           "  /a HIVE_FILE          - Analyze a registry hive.\n"
```

Comment:
Currently the `analyze` option is interactive (waiting for user input), you might consider a `batch` mode or something like that, so that people can run it non-interactive.